### PR TITLE
feat(T721-35): display notes in react app

### PR DESCRIPTION
**Pull request by TMoynat**

### CODE REVIEW

File: website/src/components/Note.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    {"⚙️"}
+                </Button>
+            </div>
+        );
+    }
+}
+
+class NoteList extends React.Component<{}, NoteState[]> {
+    componentDidMount() {
+        axios.get(`http://localhost:3001/notes`)
```
Comment (mortimr) [inline website/src/components/Note.tsx]: Shouldn't you used your environment variables here ?

File: website/src/components/Note.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+let len = 0;
+
+class Note extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props);
+        this.click = this.click.bind(this);
+    }
+    click() {
+        const noteBody = prompt('Edit note ' + this.props.id);
+        axios.put("http://localhost:3001/notes", { 'id': this.props.id, 'body': noteBody })
```
Comment (mortimr) [inline website/src/components/Note.tsx]: Same here

File: website/src/components/CreateNote.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import Button from "react-bootstrap/Button";
+
+class CreateNote extends React.Component {
+    constructor(props: any) {
+        super(props);
+        this.click = this.click.bind(this);
+    }
+    click() {
+        const noteBody = prompt('Please enter your note');
+        axios.post("http://localhost:3001/notes", { 'body': noteBody })
```
Comment (mortimr) [inline website/src/components/CreateNote.tsx]: Same here

File: website/src/components/Note.tsx (hunk 1 above)
Comment (TMoynat) [reply website/src/components/Note.tsx]: Should be fixed in latest commit

**Pull request merged**